### ISSUE TITLE
Folders to organize many OTP-entries

### ISSUE THREAD

**Issue by shamrockDE** [matteo-convertino/otpmanager-nextcloud#12]
# Feature request

## Problem
My OTP-Entries are increasing rapidly, as more and more websites support 2FA.

## Solution
In order to better organize the entries, the use of folders would be very helpful.

**Comment by matteo-convertino** [comment]
Hi, thanks for the suggestion. This is a feature that is also in the [planned features ](https://github.com/matteo-convertino/otpmanager-app#planned-features)of the app. Bringing it here on the web app is not a bad idea. 
However, I would like to say that unfortunately it is not a feature that can be developed by me in the short term.

**Comment by Bytrider** [comment]
Hello,

First of all, thank you very much for the continuous development of the app.

We are using the Nextcloud OTP app more and more, and it has become an important part of our daily workflow. However, as the number of entries continues to grow, it is becoming increasingly difficult to keep everything organized.

We would really appreciate further improvements, especially the ability to organize entries into groups or folders. Support for tags would also be a great addition and would make managing a large number of OTP entries much easier.

Thank you again for your great work and for continuously improving the app!

**Comment by matteo-convertino** [comment]
Hi, thanks to all of you who use the app.

I was thinking about groups or tags, but I wondered: how useful are they in reality?

I don’t know about you, but whenever I need an OTP code, I type the name/issuer into the search bar and copy the code. Grouping them together seems to me to be more of a ‘visual’ need than a ‘practical’ one.

**Comment by mebellin** [comment]
> I was thinking about groups or tags, but I wondered: how useful are they in reality?

I think groups would be useful if they can be shared. Sharing a group lets teams share multiple OTP accounts at once.

**Comment by Bytrider** [comment]
I agree that this is primarily a visual issue. However, @mebellin is also right that this is certainly a significant advantage when multiple OTPs need to be shared with one person or group. Since we already have some group logins using OTPs, this would considerably simplify sharing them with others.

**Comment by ivoruetsche** [comment]
Would be very nice to have folders, maybe subfolders and sharing. In the meantime, we have a lot of 2FA's and it would be nice to organize them and share it with some team members.
